data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1843 @@
+				`	(Try and dig through the snowy mounds to find something.)`
+			label dig
+			`	You claw away at the snow and hit a hard surface, but it turns out to be just a rock. You try some other mounds to no avail, and right as you finish digging through another pile of snow only to find another rock, Iplora appears behind you.`
+			`	"Counterproductive, cleaning away the snow is," she says. "Found nothing, I have. Back to your ship, we must go."`
+			choice
+				`	(Go back to the ship.)`
+					goto back
+				`	(Keep digging.)`
+					goto keepdig
+			label shelter
+			`	You rush inside the nearest cave, waiting for the weather to clear up a bit. After a while you notice a light coming from further inside the cave, and Iplora emerges soon after. "Found anything?" she asks, and you shake your head. Aside from some small campsites, she didn't either. You wait inside a bit more, and when the weather clears and night approaches, you follow her back to the ship.`
+			`	Once you're there, you leave to pick up Kirrie and Tugroob, who also come back empty handed.`
+				accept
+			label bumpyroad
+			`	Following her up the hills proves difficult at first, with your legs sinking into the snow, but after a few minutes the ground below becomes more and more solid - if a bit bumpy. Iplora explains that the area has several small and medium sized boulders buried by the snow over the years. You try to watch your step, but as the snowfall grows stronger, you trip over one of the mounds. She helps you up, and the two of you eventually arrive at the entrance of one of the caverns.`
+			choice
+				`	(Go inside with Iplora.)`
+					goto cave
+				`	(Search around the snow field.)`
+			`	You tell her you'll look around the snow for a while and go inside if the storm gets rough again. Walking around in the blizzard, this time minding the bumpy ground below, you try to spot anything unusual in the snow. Although the scenery is a great sight, the task itself quickly becomes dull. Even after walking far away from the hills, your effort produces no results. The snow starts falling much faster, so you begin making your way back to the caverns.`
+			`	With all the snow blocking your vision, you end up tripping over some of the mounds on your way and realize you don't exactly remember which cave Iplora entered. Considering the weather, though, taking shelter in any of them would be a good choice.`
+			choice
+				`	(Enter the nearest cavern.)`
+					goto shelter
+				`	(Try and dig through the snowy mounds to find something.)`
+					goto dig
+			label cave
+			`	Iplora brings out a flashlight and lights up the way as you two explore the caverns. After some twists and turns she notices some large rocks, broken free from the walls, but still hugging them closely. She asks that you hold the flashlight for her and starts pushing the rocks, revealing a passage to a small campsite containing some makeshift beds, small wooden crates, and a worn down campfire in the middle.`
```

Comment:
The wooden crates need less explanation here, but still, refer back to advanced society reference. Having them used at all could bear some explanation.